WireAuthentication/Sources/WireAuthenticationUI/Views/DetermineAuthMethod/DetermineAuthMethodView.swift

```diff
@@ -107,6 +111,17 @@ package struct DetermineAuthMethodView: View {
                 Color.red
             }
         }
+        .sheet(
+            item: modalDestination,
+            onDismiss: {
+                modalDestination.wrappedValue = nil
```

Comment:
question: is this line necessary? Won't it automatically nil out the value?

Reply:
removed code